web/beacon-app/src/features/projects/components/APIKeysTable.tsx

```diff
@@ -2,9 +2,17 @@ import { Heading, Table, Toast } from '@rotational/beacon-core';
 import { useEffect, useState } from 'react';
 
 import { ApiKeyModal } from '@/components/common/Modal/ApiKeyModal';
+import ConfirmedIndicator from '@/components/icons/confirmedIndicator';
+import PendingIndicator from '@/components/icons/pendingIndicator';
+import RevokedIndicator from '@/components/icons/revokedIndicator';
+import UnusedIndicator from '@/components/icons/unusedIndicator';
```

Comment:
Just a suggestion. What do you think if you add `Icon` as suffix to differentiate Icon to components ?

Reply:
I've updated the file name.